config/rbac/role.yaml

```diff
@@ -0,0 +1,88 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
+rules:
+- apiGroups:
```

Comment:
Maybe miss the hotspotRules?

Reply:
This was generated via Makefile. Seems there is something missing regarding the configuration of hotspot rules?